doc/ale.txt

```diff
@@ -317,7 +318,7 @@ Notes:
 * API Blueprint: `drafter`
 * AsciiDoc: `alex`!!, `proselint`, `redpen`, `write-good`
 * Awk: `gawk`
-* Bash: `shell` (-n flag), `shellcheck`, `shfmt`
+* Bash: `language-server`, `shell` (-n flag), `shellcheck`, `shfmt`
```

Comment:
Add this to `README.md` too. Have a look at the output from Travis, which complains about some things like this.

Reply:
done in a7ec36d